Quaver.Shared/Screens/Gameplay/Rulesets/HitObjects/HitObjectManager.cs

```diff
@@ -155,8 +183,12 @@ public static void PlayObjectKeySounds(HitObjectInfo hitObject)
         /// <param name="info"></param>
         /// <param name="timingPoint"></param>
         /// <returns></returns>
-        public static int GetBeatSnap(HitObjectInfo info, TimingPointInfo timingPoint)
+        public static int GetBeatSnap(HitObjectInfo info, TimingPointInfo timingPoint, bool isEditorObject = false)
```

Comment:
Might make sense to rename this to something like `GetBeatSnapIndex` or something that relates to the index used for hitobject textures, since I think "BeatSnap" implies the return result being 1,2,3,4,6,8,12,16,48